controllers/mysql_combined_test.go

```diff
@@ -76,6 +81,8 @@ func TestMySQLHappyPath(t *testing.T) {
 
 	ruleName := GenerateTestResourceNameWithRandom("mysql-fw", 10)
 
+	// This rule opens access to the public internet, but in this case
+	// there's literally no data in the database
```

Comment:
Yeah, I originally tried to get the IP and then realized it wasn't going to work because we dont know what routing looks like outside of the docker container/VM.